borg/paperkey.html

```diff
@@ -2217,7 +2220,7 @@
     <div id="printout">
         <div id="title" contenteditable>BorgBackup Printable Key Backup</div>
         <div contenteditable>To restore either scan the QR code below, decode it and import it using
-<pre>borg key-import /path/to/repo scannedfile</pre>
+<pre>borg key import /path/to/repo scannedfile</pre>
```

Comment:
imho clear enough that with "scan the QR code" using a QR scanner is meant not an imaging scanner. In the former case you'll get text / text file which can be directly consumed here. (After copying to PC).